docs/book/src/developer/architecture/controllers.md

```diff
@@ -14,6 +14,8 @@ Documentation for the CAPI controllers can be found at:
   - [MachineDeployment](./controllers/machine-deployment.md)
   - [MachineHealthCheck](./controllers/machine-health-check.md)
   - [MachinePool](./controllers/machine-pool.md)
```

Comment:
The page also has to be added to `SUMMARY.md`, e.g. like:
```
- [ClusterTopology](./developer/architecture/controllers/cluster-topology.md)
```
Otherwise mdbook won't render the page

fyi: You can render the book locally via `make serve-book` (localhost:3000)